### ISSUE TITLE
Plz release new version to Nuget

### ISSUE THREAD

**Issue by atsapura** [microsoft/fsharplu#105]
It'd be nice to have [this changes](https://github.com/microsoft/fsharplu/commit/b663c9219c65cadc63088e2f44c40d6b1806ce56) available

**Comment by blumu** [comment]
@atsapura There is PR !104 for this release. It's pending on a review. I'll ping people to get it approved soon.

**Comment by atsapura** [comment]
@blumu cool. Is there anything i can do to speed things up?

**Comment by blumu** [comment]
@atsapura I should be able to grant you write access so you can review the PR provided that you've already signed the CLA: https://cla.opensource.microsoft.com/microsoft/fsharplu

**Comment by blumu** [comment]
@atsapura The changes are now released on Nuget at https://www.nuget.org/packages/Microsoft.FSharpLu.Json/0.11.7
Thanks again for your contribution! And thanks to @ScottArbeit for reviewing and approving the release! 👏👏
